### ISSUE TITLE
Receiving an error when trying to vote on an issue

### ISSUE THREAD

**Issue by jason-entenmann** [gov4git/desktop-application#10]
I attempted to submit a positive vote on 'A Second Issue'. After clicking submit on the vote, I received the following error:

Message: Failed to cast vote. Error: Command failed: C:\Users\jentenmann\AppData\Local\Programs\gov4git-desktop-app\resources\app.asar.unpacked\node_modules\@gov4git\js-client\bin\gov4git.exe ballot vote --name github/issues/31 --choices prioritize --strengths 100 --config C:\Users\jentenmann\.gov4git\e3e675a6479a9540bd8c79c88020d4954665c5f0b179e79515cb43ae7bc455e2.json cannot find user with id 7btcyz2m7dwzsyan4xbqi3igumwdyq7n5wsqg4ydmwvtmt7jze2q in the community

Attaching error logs.

[error log 102423.txt](https://github.com/gov4git/desktop-application/files/13127838/error.log.102423.txt)

**Comment by jason-entenmann** [comment]
Desktop app Version: 0.1.1

**Comment by jason-entenmann** [comment]
After chatting with Derek, I restarted the app in order to trigger an app update to v0.2.1. With the app updated, I attempted to re-cast the same vote but still failed. see below.

Message: Failed to cast vote. Error: Command failed: C:\Users\jentenmann\AppData\Local\Programs\gov4git-desktop-app\resources\app.asar.unpacked\node_modules\@gov4git\js-client\bin\gov4git.exe ballot vote --name github/issues/31 --choices prioritize --strengths 100 --config C:\Users\jentenmann\.gov4git\e3e675a6479a9540bd8c79c88020d4954665c5f0b179e79515cb43ae7bc455e2.json cannot find user with id 7btcyz2m7dwzsyan4xbqi3igumwdyq7n5wsqg4ydmwvtmt7jze2q in the community


[error log-2 102323.txt](https://github.com/gov4git/desktop-application/files/13127932/error.log-2.102323.txt)
